webtau-groovy/src/main/resources/examples/graphql/introspection.groovy

```diff
@@ -0,0 +1,8 @@
+import static org.testingisdocumenting.webtau.WebTauGroovyDsl.*
+
+scenario('fetch types from schema') {
+    def query = '{ __schema { types { name } } }'
+    graphql.execute(query) {
+        __schema.types.numberOfElements().shouldBe > 0
```

Comment:
is it worth asserting more details here? Or is the main point that you're getting something meaningful back?

Reply:
The latter really.  I can't assert much more because I'm expecting them to point to their own GraphQL server and so I have no idea what types to expect.